easybuild/framework/easyconfig/default.py

```diff
@@ -152,6 +152,7 @@
     'hiddendependencies': [[], "List of dependencies available as hidden modules", DEPENDENCIES],
     'multi_deps': [{}, "Dict of lists of dependency versions over which to iterate", DEPENDENCIES],
     'multi_deps_load_default': [True, "Load module for first version listed in multi_deps by default", DEPENDENCIES],
+    'multi_deps_extensions_only': [False, "Use multiple dependencies for extensions only", DEPENDENCIES],
```

Comment:
I would make this `"Use dependencies specified via multi_deps only to install extensions"`

Reply:
That was too long for a single line. I went with a compromise.